src/containers/AccountModal/forms/ChooseOffer.tsx

```diff
@@ -12,50 +12,98 @@ import { getOffer } from '#src/services/checkout.service';
 import LoadingOverlay from '#src/components/LoadingOverlay/LoadingOverlay';
 import ChooseOfferForm from '#src/components/ChooseOfferForm/ChooseOfferForm';
 import useForm, { UseFormOnSubmitHandler } from '#src/hooks/useForm';
-import type { ChooseOfferFormData, OfferPeriodicity } from '#types/account';
+import type { ChooseOfferFormData, OfferPeriodicity, OfferType } from '#types/account';
+import type { Offer } from '#types/checkout';
 
 const ChooseOffer = () => {
   const history = useHistory();
   const { t } = useTranslation('account');
-  const { config, accessModel } = useConfigStore(({ config, accessModel }) => ({ config, accessModel }), shallow);
+  const { config } = useConfigStore(({ config }) => ({ config }), shallow);
```

Comment:
I guess destructuring isn't necessary anymore, but not a big deal since `config` never changes.

```suggestion
  const config = useConfigStore(({ config }) => config);
```